activerecord/lib/active_record/enum.rb

```diff
@@ -75,6 +75,22 @@ module ActiveRecord
   #
   #   Conversation.where("status <> ?", Conversation.statuses[:archived])
   #
+  # You can use <tt>:enum_prefix</tt>/<tt>:enum_suffix</tt> option then you need
```

Comment:
Right, `when` would be better, since after this word you have explanation for its usage.